reinforcement_learning/rl_portfolio_management_coach_customEnv/rl_portfolio_management_coach_customEnv.ipynb

```diff
@@ -324,8 +324,8 @@
     "                      toolkit_version='0.11.0',\n",
```

Comment:
<p>Is the Coach 1.0 version not working for this notebook (and others that are still using 0.11)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1706/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
This example uses MxNet and there's no docker image for Coach 1.0 + MxNet. For all TF notebooks the Coach version was updated to 1.0.